properties/base.py

```diff
@@ -190,13 +215,42 @@ def _notify(self, change):
     def _set(self, name, value):
         change = dict(name=name, value=value, mode='validate')
         self._notify(change)
-        if change['value'] is utils.undefined and name in self._backend:
-            self._backend.pop(name)
+        if change['value'] is utils.undefined:
+            self._backend.pop(name, None)
```

Comment:
I like `.pop`. Is it for sure available in all backends? May want to add a set of "need to implement these to write a backend" notes. https://github.com/3ptscience/properties/issues/88